assets/component-newsletter.css

```
@@ -16,15 +16,19 @@
 }
 
 .newsletter-form__field-wrapper {
-  width: 100%;
-}
-
-.newsletter-form__field-wrapper .field__input {
-  padding-right: 5rem;
+  width: max-content;
 }
 
 .newsletter-form__field-wrapper .field {
   z-index: 0;
+  display: flex;
```

```suggestion
  display: flex;
  align-items: center;
```

Just noticed a slight misalignment with the input and button that is fixed with this change.